stripe/src/main/java/com/stripe/android/model/StripeIntent.kt

```diff
@@ -192,7 +193,7 @@ interface StripeIntent : StripeModel {
     }
 
     companion object {
-        internal sealed class NextActionData : StripeModel {
+        sealed class NextActionData : StripeModel {
```

Comment:
Should make the classes that extend this public too or else nobody will actually be able to use this object